.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
```

Comment:
what does this part do?

Reply:
husky puts that in. Their instructions to create a hook are `npx husky add .husky/pre-commit "npm test"` and it auto generates that